packages/wallets/src/features/cashier/modules/WithdrawalCrypto/provider/WithdrawalCryptoProvider.tsx

```diff
@@ -90,6 +93,12 @@ const WithdrawalCryptoProvider: React.FC<React.PropsWithChildren<TWithdrawalCryp
         return () => unsubscribe();
     }, [activeWallet?.currency, activeWallet?.loginid, subscribe, unsubscribe]);
 
+    useEffect(() => {
+        if (isCryptoConfigError) {
+            setError(cryptoConfigError?.error);
+        }
+    }, [cryptoConfigError, isCryptoConfigError]);
```

Comment:
could we just rely on error value? and don't not check  `isCryptoConfigError`?

Reply:
+1

Reply:
Updated